packages/slurp-cohttp/slurp-cohttp.0.1.0/opam

```diff
@@ -0,0 +1,37 @@
+opam-version: "2.0"
+synopsis: "Cohttp definition for Slurp"
+maintainer: "Julien Blond <julien.blond@gmail.com>"
+authors: "Julien Blond <julien.blond@gmail.com>"
+license: "Apache 2.0"
+tags: ["sinatra" "route" "http" "server" "cohttp"]
+homepage: "https://github.com/Ninjapouet/slurp"
+bug-reports: "https://github.com/Ninjapouet/slurp/issues"
+depends: [
+  "dune" {>= "2.7"}
+  "slurp" {>= "0.1.0"}
```

Comment:
If you intend in releasing `slurp` and `slup-cohttp` together, I would suggest to change this constraint to `{= version}` as it will simplify future releases and avoid breakage of the previous versions.

Reply:
Why being so restrictive the breakage will most likely append by other packages introduced by (>=) property; why being so restrictive ? Although the two packages are effectively intended to be released together, i often encountered the case where the binary was kept at version X but library at version Y > X  for the lib Y fixed some security issue but the binary could not be changed because of other lib dependency reasons.